cmd/minikube/cmd/service.go

```diff
@@ -45,6 +46,14 @@ var (
 	interval           int
 )
 
+// type serviceNotFoundError struct {
```

Comment:
Remove commented-out code

Reply:
Sorry for this. I have removed the same. Thank you! :)